tf_pip/tf212_env.yml

```diff
@@ -4,11 +4,11 @@ channels:
   - nvidia
 dependencies:
   - python=3.10
-  - numpy=1.23.*
-  - cudatoolkit=11.8.0
+  - libblas=*=*mkl
+  - numpy
```

Comment:
Will numpy install properly if you remove the 1.23.* designation? The pip installation of tensorflow downgrades numpy but then does not downgrade scipy or pandas, resulting in errors.